samples/src/main/java/com/example/bigquery/RelaxTableQuery.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigquery;
+
+// [START bigquery_relax_column_query_append]
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.bigquery.JobId;
+import com.google.cloud.bigquery.JobInfo;
+import com.google.cloud.bigquery.JobInfo.SchemaUpdateOption;
+import com.google.cloud.bigquery.JobInfo.WriteDisposition;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableId;
+import com.google.cloud.bigquery.TableResult;
+import com.google.common.collect.ImmutableList;
+import java.util.UUID;
+
+public class RelaxTableQuery {
+
+  public static void runRelaxTableQuery() throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "MY_PROJECT_ID";
+    String datasetName = "MY_DATASET_NAME";
+    String tableName = "MY_TABLE_NAME";
+    relaxTableQuery(projectId, datasetName, tableName);
+  }
+
+  public static void relaxTableQuery(String projectId, String datasetName, String tableName)
+      throws Exception {
+    try {
```

Comment:
Do BigQuery samples usually not use the java `try-with-resources?

Like:
```
try (BigQuery bigquery = BigQueryOptions.getDefaultInstance().getService()) {
...
```
https://docs.oracle.com/javase/7/docs/technotes/guides/language/try-with-resources.html

Reply:
That's right -- BQ client is not Autocloseable and cannot be changed into one since there's no close() method we can call to clean up resource. The Apiary client generator would have to change... something I discussed with Les and will put on hold for now.

Reply:
The BQ client is hand written and has a few quirks - one is the client isn't autoclosable.